cmd/paasta/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+func listPaastaCommands() (map[string]bool, error) {
```

Comment:
nit: `map[string]bool` type is unclear.  `string` is probably a command name, but what is `bool` for?  Maybe introduce a type with name like `CommandNameToSomething`?

Reply:
`map[string]bool` is the way you get "sets" in golang because it's almost 2020 and almost trillion dollar company doesn't know a thing about programming languages. i'll add a comment.